lib/utils/validateRequired.js

```diff
@@ -35,3 +35,19 @@ export function validateRequiredPositiveAmount(value) {
     ? undefined
     : <FormattedMessage id="stripes-acq-components.validation.shouldBePositiveAmount" />;
 }
+
+export function validateRequiredMinAndMaxNumber({ minNumber, maxNumber, value }) {
```

Comment:
I'd suggest having two separate validation functions for min and max numbers instead. This way functions have a single responsibility and are easier to reuse